sway/config/output.c

```diff
@@ -359,6 +372,7 @@ static void default_output_config(struct output_config *oc,
 	}
 	oc->x = oc->y = -1;
 	oc->scale = 1;
+	oc->subpixel = WL_OUTPUT_SUBPIXEL_UNKNOWN;
```

Comment:
Oh sorry, missed this. Either use:
```c
struct sway_output *output = wlr_output->data;
oc->subpixel = output->detected_subpixel;
```
or change the function signature to take a `struct sway_output` instead of `struct wlr_output` and use
```
oc->subpixel = output->detected_subpixel;
```